contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -3840,7 +3841,13 @@ gen_sp_rename_subcmds(const char *objname, const char *newname, const char *sche
 				 errmsg("Provided objtype is not supported for sp_rename")));
 			break;
 	}
+	old_dialect = sql_dialect;
+	sql_dialect = SQL_DIALECT_PG;
+	/* this query must be run in PG dialect or we will get a syntax error due to different
+	 * ALTER behavior between PG and TSQL
```

Comment:
Which test case is corresponding to this code ?

Reply:
Just spoke with Jason - this is required to avoid a syntax error when parsing alter proc. If we remove the code there is a syntax error